### ISSUE TITLE
Support "@default":[] in framing

### ISSUE THREAD

**Issue by amiika** [jsonld-java/jsonld-java#143]
Default value in framing is not working as expected. Example that is working in JSON-LD Playground but not with JSONLD-Java: http://tinyurl.com/pwexaa6

**Comment by hmottestad** [comment]
I just ran into this as well.

**Comment by ansell** [comment]
Should be fixed by #167, if not reopen this issue.

**Comment by hmottestad** [comment]
@amiika This frame works better in the json-ld playground:

```
{
  "@context": {
    "ex": "http://example.org/ex#",
    "foaf": "http://xmlns.com/foaf/0.1/",
    "owl": "http://www.w3.org/2002/07/owl#",
    "rdf": "http://www.w3.org/1999/02/22-rdf-syntax-ns#",
    "rdfs": "http://www.w3.org/2000/01/rdf-schema#",
    "xsd": "http://www.w3.org/2001/XMLSchema#",
    "frapo": "http://purl.org/cerif/frapo/",
    "foaf:pastProject": {
      "@id": "foaf:pastProject",
      "@container": "@set"
    },
    "foaf:currentProject": {
      "@id": "foaf:currentProject",
      "@container": "@set"
    }
  },
  "@type": "foaf:Organisation",
  "foaf:member": {
    "foaf:currentProject": {
      "@default": "@null",
      "@embed": false
    },
    "foaf:pastProject": {
      "@default": "@null",
      "@embed": false
    }
  },
  "frapo:funds": {}
}
```
